go-controller/pkg/node/management-port.go

```diff
@@ -70,6 +70,12 @@ func newManagementPort(nodeName string, hostSubnets []*net.IPNet) ManagementPort
 }
 
 func (mp *managementPort) Create(nodeAnnotator kube.Annotator, waiter *startupWaiter) (*managementPortConfig, error) {
+	for _, mgmtPortName := range []string{types.K8sMgmtIntfName, types.K8sMgmtIntfName + "_0"} {
```

Comment:
> it would be nice if types held a constant for VF representor name. ...

I'd prefer to not, since in the future we suppose to add (to upstream) support for workload pods on the DPU. In approach we do have now there are 2 instances of ovnkube-node on the DPU (arm host) - full mode (using SF and its representor) and dpu mode (using VF representor). Within such approach representors must have different names. 

> The code is confusing around various places in the code, but it looks like the VF representor name should always be ovn-k8s-mp0_0. Is that right?
> ...
> Is the representor different names depending on OVNK full vs dpu mode?

Yes, it is. I kept original name for DPU mode.

Reply:
Is there a predictable naming scheme in each of these scenarios? Or will the names sometimes be random? Might be good to have some readme document explaining how the naming conventions work.

Reply:
No. Names are predictable. In most cases it is `ovn-k8s-mp0`. Only in case of running full mode it is `ovn-k8s-mp0` and `ovn-k8s-mp0_0`. In case of 2 instances of ovnkube-node running on arm host, I mentioned above, we are planning to have mgmt port name in some way configurable. In approach we do have, we just pass mgmt port name as environment variable for full mode ovnkube-node. So all mgmt port interfaces will look like:

```
# ip -br link  | grep ovn
ovn-k8s-mp1_0    ...  # full mode SF representor
ovn-k8s-mp1      ...  # full mode SF
ovn-k8s-mp0      ...  # dpu mode VF representor
```